pkg/datasource.go

```diff
@@ -318,8 +319,13 @@ func (o *OCIDatasource) regionsResponse(ctx context.Context, req *backend.QueryD
 
 		frame := data.NewFrame(query.RefID, data.NewField("text", nil, []string{}))
 
-		for _, item := range res.Items {
-			frame.AppendRow(*(item.Name))
+		var regions []string
+		for _, regionName := range res.Items {
```

Comment:
My comment related to the use of the `item` variable name was only about the for loop starting on line 327. Here the `item` variable name makes sense since the variable being looped over is `res.Items`.  It is up to you whether you go back and revert this variable name change.

Reply:
I've approved the PR but if you go back and revert this variable name then I'll re-approve it.